src/App.tsx

```
@@ -1,23 +1,53 @@
+    } else {
+      html.setAttribute("data-theme", "dark");
+      localStorage.setItem("theme", "dark");
+      setTheme("dark");
+    }
+  };
 
   return (
     <>
-      <div>
+      <div style={{ display: "flex", gap: "1rem", alignItems: "center" }}>
         <a href="https://vite.dev" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)